tests/test-util/src/math.spec.ts

```diff
@@ -22,4 +22,8 @@ describe("math", function () {
         expect(normalize(1000, 0, 365)).to.equal(270);
         expect(normalize(1000, -100, 100)).to.equal(0);
     });
+    it("pointInPolygon", function () {
+        expect(pointInPolygon([1, 1], [[0, 0], [0, 8], [8, 8], [8, 0]])).to.equal(true);
+        expect(pointInPolygon([9, 9], [[0, 0], [0, 8], [8, 8], [8, 0]])).to.equal(false);
```

Comment:
Code looks good - can you add some extra tests for when the point is:
* Outside only on one axis
* Is on the line / point of the polygone (is that in or out?)